lib/web/mage/adminhtml/wysiwyg/tiny_mce/plugins/magentowidget/editor_plugin.js

```diff
@@ -170,7 +171,7 @@ define([
                         var attributes = wysiwyg.parseAttributesString(match[2]),
                             widgetCode;
 
-                        if (attributes.id) {
+                        if (attributes.id && Base64.isBase64(attributes.id)) {
                             widgetCode = Base64.idDecode(attributes.id);
```

Comment:
Hi @benathomeofchampions thank you for your contribution, after small fixes where you forgot to add your method in min.js file, we found that tests fail with insert widget in WYSIWYG edit could you please see failed test and fix them? the problem is that when you want to add 2 widgets with catalog product list the second widget is not decoding.
![DeepinScreenshot_select-area_20191009110735](https://user-images.githubusercontent.com/51681547/66463093-1081c200-ea85-11e9-885c-d68b5dd62826.png)